src/twisted/newsfragments/9972.feature

```diff
@@ -0,0 +1 @@
+Support `yield` on coroutines in `inlineCallbacks` functions.
```

Comment:
I don't understand this sentence. Sorry.

Maybe something like this ?

```suggestion
twisted.internet.defer.inlineCallbacks can now yield a coroutine.
```

Reply:
Thanks, this indeed sounds better.